apps/client/src/metrics/metric-handler.ts

```diff
@@ -0,0 +1,49 @@
+import { onCLS, onFID, onFCP, onLCP, onTTFB, Metric } from 'web-vitals';
+
+class MetricHandler {
+  private queue = new Set<Metric>();
+
+  enqueue = (metric: Metric) => {
```

Comment:
Prefer using method declarations instead of arrow functions in classes. This is not truly an issue here, as there is only one instance, but you otherwise you lose the performance benefits of classes when there are many objects due to instantiation of the arrow functions in every class. I would prefer we don't need to consider this difference.

Reply:
Thanks for the feedback, I agree that it's a good practice in general. In this case I'm actually avoiding unintentional scopings of `this` by using arrow functions, which I think simplifies the code (avoiding hard bindings). I could update `sendMetrics` to use a method declaration but I'd prefer to leave it for now because it's not implemented and the class is exposed (like you said) as a singleton.